spec/rack/test/cookie_jar_spec.rb

```diff
@@ -67,4 +67,17 @@
     jar.merge('c=d; domain=example.org; secure', URI.parse('/'))
     jar.to_hash.must_equal 'a' => 'b'
   end
+
+  it '#merge ignores empty cookies in cookie strings' do
+    jar = Rack::Test::CookieJar.new
+    jar.merge('', URI.parse('/'))
+    jar.merge("\nc=d")
```

Comment:
oh hey, I missed a URI second arg here. Maybe that should be added if it's needed.